site/docs/v1/tech/events-webhooks/events.adoc

```diff
@@ -38,10 +38,9 @@ The ability to limit the generation of an event for only certain applications is
 
 These events can be application scoped:
 
-* `user.action`
 * `jwt.refresh-token.revoke`
-* `jwt.refresh`
 * `jwt.public-key.update`
+* `user.action`
```

Comment:
Any reason not to alphabetize? (same for list below as well)

Reply:
No reason, will do that.